### ISSUE TITLE
Smoke Test fails on master

### ISSUE THREAD

**Issue by MaoZiming** [skypilot-org/skypilot#2773]
`test_gcp_storage_mounts`, `test_azure_http_server_with_custom_ports`, `test_gcp_storage_mounts_with_stop` fail on master.

**Comment by Michaelvll** [comment]
cc'ing @romilbhardwaj  and @landscapepainter

**Comment by landscapepainter** [comment]
@Michaelvll Confirmed that `test_gcp_storage_mounts` and `test_gcp_storage_mounts_with_stop` are passing from #2322

**Comment by romilbhardwaj** [comment]
Thanks for reporting @MaoZiming. Dug into it, looks like it's stemming from the cleanup of the test (the test itself is successful):
```
FAILED tests/test_smoke.py::test_gcp_storage_mounts - subprocess.CalledProcessError: Command 'sky down -y t-gcp-storage-mounts-89; sky storage delete sky-test-1699772876' returne...
```
Looks like this was introduced by #2726 - our tests were not updated to include`-y` for `sky storage delete` calls. Will submit a PR.

**Comment by Michaelvll** [comment]
This should be fixed by #2774. Closing.
